STEER/ESD/AliESDpid.cxx

```diff
@@ -36,6 +36,7 @@
 #include "AliESDtrack.h"
 #include "AliMCEvent.h"
 #include "AliTOFPIDParams.h"
+#include "AliPID.h"
```

Comment:
Is this include needed? It was not there before, and AliPID was used.

Reply:
If the include was coming from other dependencies, its explicit inclusion will not do any harm, so, it is ok

Reply:
Yes, sure, I would have avoided it if not needed, but I totally agree. I was curious to know why @mcolocci added it.

Reply:
Ciao @chiarazampolli. About the AliPID dependence could be indeed avoided. I added at the beginning because we were seeing an error during the rebuild and at the first glance we thought they were related to it. Then, when I managed to rebuild fixing the error who was somewhere else…, I simply forgot to remove it. Thanks for checking!